src/appBootstrapper.tsx

```diff
@@ -235,6 +237,26 @@ if (__DEBUG__ && module.hot) {
     module.hot.accept('./routes', () => render());
 }
 
+async function loadCustomJs() {
+    const customJsFiles = getServerConfig().custom_js_files;
+    if (!customJsFiles) {
+        return Promise.resolve();
+    }
+    getServerConfig().customJsFunctions = {};
+    return Object.values(customJsFiles).map(
+        (customJsFile: CustomJsFile) =>
+            new Promise((resolve, reject) => {
+                load(customJsFile.path, (err: any) => {
```

Comment:
does ```load``` not return a promise directly?  in any case, we should probably show a big alert message or something if one of these fails to load.

Reply:
Shouldn't those custom js files be optional? 
So if it fails to load anything, the default behaviour will kick in I guess?